tools/skywalker_diff.py

```diff
@@ -0,0 +1,89 @@
+# This script generates norms for cross-validating the particle nucleation
+# process. To generate the data needed by the script, run skywalker in the
+# current directory, using nucleation.yaml as input. This writes a file called
+# haero_skywalker.py to the directory, which is then imported as a module.
+# Have another haero_skywalker.py created previously.  This script
```

Comment:
This implies that both modules would be named `haero_skywalker`, right? Maybe we want to talk about `mam_skywalker` or `boxmodel_skywalker` vs `haero_skywalker` instead?

Reply:
Yes, if someone wants to compare the Skywalker results over time or between MAMRefactor and Hearo the results files will need to be organized to have different names or different directories.   I think this is the standard problem of organizing output files for comparison.  I tried to clarify by updating the comment.